package.json

```diff
@@ -21,15 +23,17 @@
   },
   "devDependencies": {
     "chai": "^3.5.0",
-    "mocha": "^2.4.5"
+    "mocha": "^2.4.5",
+    "babel-cli": "^6.18.0",
+    "babel-preset-es2015": "^6.18.0"
   },
   "dependencies": {
     "argparse": "1.0.7",
     "async": "2.0.1",
     "bunyan": "1.8.1",
     "md5": "2.1.0",
     "moment": "2.12.0",
-    "ros_msg_utils": "RethinkRobotics-opensource/ros_msg_utils#v1.0.1",
+    "ros_msg_utils": "chfritz/ros_msg_utils#add_babel",
```

Comment:
`ros_msg_utils` has been published so you should be able to set this to 1.0.2 - I think that's the only thing left to update